lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -50,6 +50,24 @@
         </label>
     %endif
     <div class="enroll-option">
+        <label for="course-modes-list-selector" class="has-hint">${_("Select a course mode: ")}
+            <select id="course-modes-list-selector" name="course_mode" class="course-modes-list-selector">
+                %for mode_name, mode_data in section_data['course_modes'].items():
+                    <option value="${mode_name}">${mode_data.name}</option>
+                %endfor
+            </select>
+
+            <div class="hint dropdown-hint auto-enroll-hint">
+                <span class="hint-caret"></span>
+                <p id="auto-enroll-tip">
+                    ${_("The users will be automatically enrolled in the <em>selected</em> course mode.")}
```

Comment:
Learners will be automatically enrolled in the course mode that you select.

(Note no `<em>` tags.)